src/views/Claiming/Claiming.tsx

```diff
@@ -189,6 +190,13 @@ export const Claiming: React.FC = () => {
     return (
       <>
         <MenuSection>
+          <GroupByOrgActionMenuItem
+            onClick={(e: Event | undefined) => {
+              onToggle(e);
+              changeSorting(e, { name: CLAIMING_LIST_COLUMNS.vendorCode });
```

Comment:
CQL sorting is possible only by fields specified in the DB table.

https://s3.amazonaws.com/foliodocs/api/mod-orders/r/pieces.html#orders_pieces_get
https://statics.teams.cdn.office.net/evergreen-assets/safelinks/1/atp-safelinks.html